.stylelintrc.json

```diff
@@ -44,6 +49,7 @@
     "declaration-colon-newline-after": "always-multi-line",
     "declaration-colon-space-after": "always",
     "declaration-colon-space-before": "never",
+    "declaration-no-important": true,
```

Comment:
Nothing is ever that `!important`. If you need `!important` then more than likely you need to refactor your HTML. For the rare case of needing to override a 3rd party library rule this stylelint rule can be disabled inline.